src/NuGet.Server.V2/Controllers/NuGetODataController.cs

```diff
@@ -403,8 +403,16 @@ public virtual async Task<HttpResponseMessage> UploadPackage(CancellationToken t
             HttpResponseMessage retValue;
             if (_authenticationService.IsAuthenticated(User, apiKey, package.Id))
             {
-                await _serverRepository.AddPackageAsync(package, token);
-                retValue = Request.CreateResponse(HttpStatusCode.Created);
+                try
+                {
+                    await _serverRepository.AddPackageAsync(package, token);
+                    retValue = Request.CreateResponse(HttpStatusCode.Created);
+
+                }
```

Comment:
Please remove the empty line on line 410.

Reply:
Done!